openapi/code_samples/python/api_v1_users/get.py

```
@@ -1,2 +1,2 @@
-# pass empty org to fetch all users in environment
-resp,_ = scalekit_client.users.list_users(organization_id="", page_size=100)
\ No newline at end of file
+# List all users in the environment
+response = scalekit_client.users.list_users(page_size=100)
```

Ruff (0.15.21)

[error] Undefined name `scalekit_client`

(F821)